### ISSUE TITLE
[1.8.1] auto client refresh not working

### ISSUE THREAD

**Issue by trusktr** [meteor/meteor#10694]
Seems like auto-refresh doesn't work for me. I'm in Chrome, and I have to manually hit the refresh button to try new changes.

I do see stuff like

```
=> Client modified -- refreshing (x16)
```

in my terminal, but the page just doesn't refresh.

Here's a reproduction:

```sh
git clone --recursive git@github.com:trusktr/trusktr.io.git
cd trusktr.io
git checkout meteor-issue-10694
cd meteor-app
npm install
meteor
```

Do you get the same issue once you have that running, and you edit any client code?

In particular, I was editing code inside of `/meteor-app/client/imports/routes/index.js` and the page would not automatically refresh.

**Comment by trusktr** [comment]
On a related note, it started doing trying to auto-refresh indefinitely (though the client doesn't actually refresh):

![keeps-going](https://user-images.githubusercontent.com/297678/65218512-7a511200-da6b-11e9-8dfa-595a2fe5806a.gif)

**Comment by benjamn** [comment]
Can you try updating to the latest 1.8.2 beta? `meteor update --release 1.8.2-beta.18`

**Comment by rj-david** [comment]
I experience this only if I do `meteor npm install` while an app is running

**Comment by trusktr** [comment]
@benjamn I may give it a try later, because at the moment https://github.com/meteor/meteor/issues/10660 is in the way.

**Comment by trusktr** [comment]
I didn't get to verify if this is fixed on `1.8.2.beta.18`, because the other issues, but the problem is gone while I'm reverted back to the previous version I was using `1.6.1.4` .
